app/bundles/EmailBundle/EventListener/ReportSubscriber.php

```diff
@@ -525,8 +525,8 @@ public function onReportGraphGenerate(ReportGraphEvent $event)
                     );
                     $this->addDNCTableForEmails($queryBuilder);
                     $queryBuilder->resetQueryPart('groupBy');
-                    $counts = $queryBuilder->execute()->fetch();
-                    $chart  = new PieChart();
+                    $counts   = $queryBuilder->execute()->fetchOne();
```

Comment:
@escopecz just to verify that is the right method

Reply:
I think it should be https://www.doctrine-project.org/projects/doctrine-dbal/en/current/reference/data-retrieval-and-manipulation.html#fetchassociative based on how the `$counts` variable is used afterwards